docker/postgres/Dockerfile

```diff
@@ -25,7 +25,8 @@ COPY --chmod=0755 ./postgres/postgres.conf /etc/postgresql/postgresql.conf
 
 COPY --chmod=0777 ./postgres/pixl-db_init.sh /docker-entrypoint-initdb.d/pixl-db_init.sh
 
-COPY --chmod=0777 ./postgres/create_pixl_tbls.py /pixl/create_pixl_tbls.py
+COPY alembic /pixl/alembic
+COPY alembic.ini /pixl/alembic.ini
```

Comment:
so I think instead of this being run on postgres spinning up, this should be part of the starting of the `imaging-api`, so we can remove installing pixl core here as well.